apisix/plugins/grpc-transcode/proto.lua

```diff
@@ -29,6 +30,10 @@ local lrucache_proto = core.lrucache.new({
 local proto_fake_file = "filename for loaded"
 
 local function compile_proto(content)
+    -- clear pb state
+    pb.state(nil)
```

Comment:
Isn't it redundant? I think Line 54 is already retrieving the current pb state and setting it to nil.

Reply:
There is a default pb state that we need to clear.

Reply:
Got it.